website/docs/components/alert/05--accessibility.md

```diff
@@ -11,9 +11,11 @@ This component is conditionally conformant. That is, it is conformant when there
 
 This section is for reference only. This component intends to conform to the following WCAG success criteria:
 
-<dummy-wcag-success-criteria-list data-list="1.3.1|1.3.2|1.4.1|1.4.3|1.4.10|1.4.11|1.4.12|2.1.1|2.1.2|2.2.1|2.5.3|4.1.1|4.1.2|4.1.3">Placeholder for the WCAG Success Criteria List component - Don't delete!</dummy-wcag-success-criteria-list>
+<Doc::WcagList @criteriaList={{array 'wcag-1-3-1' 'wcag-1-3-2' 'wcag-1-4-1' 'wcag-1-4-3' 'wcag-1-4-10' 'wcag-1-4-11' 'wcag-1-4-12' 'wcag-2-1-1' 'wcag-2-1-2' 'wcag-2-2-1' 'wcag-2-5-3' 'wcag-4-1-1' 'wcag-4-1-2' 'wcag-4-1-3' }} />
```

Comment:
[suggestion] to make things simpler, why not just pass the values as `1-3-1` and prepend the `wcag-` prefix inside the component logic/code? this would make it shorter, more readable.

Reply:
I agree; since we're not using any of the failures or techniques in our documentation, I could simplify the data. 👍